app/translation/en.json

```diff
@@ -0,0 +1 @@
+"english translations"
```

Comment:
it looks like this PR also contains all of the changes from you lang change one, lets keep them separate